MatrixSDK/MXSession.m

```diff
@@ -402,6 +403,214 @@ -(void)setStore:(id<MXStore>)store success:(void (^)(void))onStoreDataReady fail
     }];
 }
 
+- (void)handleSyncResponse:(MXSyncResponse *)syncResponse
```

Comment:
This method should have an asynchronous callback indicating when it is done. See in `startWithSyncFilterId`.

Reply:
Updated